sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/Configs.java

```diff
@@ -68,9 +76,11 @@ public SslContext getSslContext() {
     }
 
     public Protocol getProtocol() {
-        String protocol = getJVMConfigAsString(PROTOCOL, DEFAULT_PROTOCOL.toString());
+        String protocol = System.getProperty(PROTOCOL_PROPERTY, firstNonNull(
+            emptyToNull(System.getenv().get(PROTOCOL_ENVIRONMENT_VARIABLE)),
+            DEFAULT_PROTOCOL.name()));
```

Comment:
given the fact that protocol name is different in v2 and v3, Kushagra did some coding here to ensure our CI works for both v2 and v3. As v3 is on a different repo with new CIs, I am not sure how much of that is still applicable but good to check.

@kushagraThapar does this break anything in the CI?

Reply:
Thanks for the review. Kushagra may know something more but I've not seen that CI is broken by the change. l'll wait to hear back from him before resolving this issue.